functions/date-time-offset/1.0/function.json

```diff
@@ -0,0 +1,113 @@
+{
+  "description": "Calculate a date, time or date with time based on a provided date in text (YYYY-MM-DD hh:mm:ss) or the current date",
+  "label": "Date/Time Offset",
+  "category": "Misc",
+  "icon": {
+    "name": "DateTimeIcon",
+    "color": "Yellow"
+  },
+  "options": [
+    {
+      "name": "customStartDate",
+      "label": "Specify your date in YYYY-MM-DD hh:mm:ss notation",
+      "meta": {
+        "type": "Text"
+      }
+    },
+    {
+      "name": "currentDate",
+      "label": "OR use the current date/time ",
+      "meta": {
+        "type": "Boolean"
+      }
+    },
+    {
+      "name": "offsetType",
+      "label": "Offset Type",
+      "meta": {
+        "type": "Select",
+        "validations": {
+          "required": true
+        },
+        "values": [
+          { "label": "Seconds", "value": "ss" },
+          { "label": "Minutes", "value": "mm" },
+          { "label": "Hours", "value": "hh" },
+          { "label": "Days", "value": "DD" },
+          { "label": "Weeks", "value": "WW" },
+          { "label": "Months", "value": "MM" },
+          { "label": "Years", "value": "YYYY" }
+        ]
+      }
+    },
+    {
+      "name": "offset",
+      "label": "offset",
+      "meta": {
+        "type": "Number",
+        "validations": {
+          "required": true
+        }
+      }
+    },
+    {
+      "name": "resultType",
+      "label": "Result type",
+      "meta": {
+        "type": "Select",
+        "validations": {
+          "required": true
+        },
+        "values": [
+          { "label": "Date", "value": "D" },
+          { "label": "Date Time", "value": "DT" },
+          { "label": "Time", "value": "T" },
+          { "label": "Unix Time stamp (epoch)", "value": "UT" }
+        ]
+      }
+    },
+    {
+      "name": "TimeZoneOffset",
```

Comment:
Little typo here where the T shouldn't be capitalised in order for the timezone offset to be filled.
Very handy function 👌 Would be great to get this approved and in the blockstore soonish @jwielinga

Reply:
Can be fixed later on